gfx/video_shader_parse.c

```diff
@@ -65,10 +65,8 @@ static const char *wrap_mode_to_str(enum gfx_wrap_type type)
       case RARCH_WRAP_MIRRORED_REPEAT:
          return "mirrored_repeat";
       default:
-         break;
+         return "???";
```

Comment:
Rather you change this back to what it was. I want the function to always return something at the end of the function.